tests/test_operation.py

```diff
@@ -636,7 +636,7 @@ def test_wires(self):
         X = qml.PauliX(0)
         Y = qml.Hermitian(p, wires=[1, 2])
         t = Tensor(X, Y)
-        assert t.wires == [[0], [1, 2]]
+        assert t.wires == [0, 1, 2]
```

Comment:
What happens in the following case? :thinking: 
```
X = qml.Hermitian(p, wires=[0,2])
Y = qml.Hermitian(p, wires=[1,3])
t = Tensor(X, Y)
t.wires == [0, 2, 1, 3]?
```

Reply:
one of the reasons we used a nested list instead of a flat list is because it makes it easier to determine which wires belonged to which op

Reply:
But this information is still fully available since a tensor observable carries the list of sub-observable as an attribute: 

nested_wire_list = [ob.wires for ob in t.obs] 

This is exactly what we are doing now in ``Device``, we are checking if the observable is a ``Tensor``, and if so we pass this nested list of wires.

Reply:
Sorry, I reread your comment and only gave a partial answer. ``t.wires`` will maintain the ordering of the sub-observables.